packages/flutter/test/widgets/scrollable_test.dart

```diff
@@ -109,6 +109,43 @@ void resetScrollOffset(WidgetTester tester) {
 }
```

Comment:
Can you add a test that taps something and verifies it is hit?

Reply:
Sure, Actually I was going to add it but got busy.